package.json

```diff
@@ -34,9 +34,9 @@
     "@metamask/eth-sig-util": "^4.0.0",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.5.0",
-    "@pushprotocol/restapi": "1.7.18",
+    "@pushprotocol/restapi": "1.7.19",
```

Comment:
I tested it out and chat reaction need 1.7.19 version to run properly, otherwise on the previous versions it user has to manual reload the chat to see their reactions on a chat.

The upgraded version of the restapi package 1.7.19 has a bug and fails to load DApp properly. So this needs to be resolved to make this PR work.

Refer to the error in AppContext.tsx and SpaceContext.tsx to see the error.

![Screenshot 2024-05-27 at 9 22 39 PM](https://github.com/push-protocol/push-dapp/assets/25508878/67993019-3089-4543-acde-884921fdb805)

Slack thread: https://pushprotocol.slack.com/archives/C05LL3YA7JA/p1716826289124139

Reply:
Issue is resolved after upgrading the restapi version to 1.7.20